account_bank_statement_import_adyen/tests/test_import_file.py

```diff
@@ -0,0 +1,132 @@
+"""Provide common base for bank statement import tests."""
+##############################################################################
+#
+#    Copyright (C) 2015 Therp BV <http://therp.nl>.
+#
+#    All other contributions are (C) by their respective contributors
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import logging
+import base64
+
+from odoo import tests
+from odoo.tests.common import TransactionCase
+from odoo.modules.module import get_module_resource
+
+
+_logger = logging.getLogger(__name__)
+
+
+@tests.tagged('post_install', 'at_install')
```

Comment:
I think the defaults are fine, so you don't need to add a `tagged` decorator.

Reply:
BTW I see why you included this file but I suggest you just include the bits we need to import a statement file in `test_import_adyen.py` and coassign Therp copyright on that file.

Reply:
I removed the unused method from the Therp test file. Is that enough or should I move the one method we use to our test file and delete the other file.

I can't remove the tagged decorator as the tests will not run for me.

Reply:
If your tests don't run without the decorator you are likely to find your modules have been installed with demo = false in ir_module_module and you need to ensure demo data is installed.
Please merge this file into the actual test file.